docs/data-management/connecting-dwh/bigquery.md

```diff
@@ -14,7 +14,13 @@ Additionally, you will need to create a data environment for Eppo to write inter
 4. Click **CREATE SERVICE ACCOUNT** in the service accounts header.
 5. Under **Service account details**, add an _account name_, _ID_, and optional _description_.
 6. Click **CREATE**.
-7. Under **Service account permissions**, add the following role: `BigQuery Job User (roles/bigquery.jobUser)`
+7. Under **Service account permissions**, add the following roles:
+    - `BigQuery Job User (roles/bigquery.jobUser)`
+      - Required
+    - `Storage Admin (roles/storage.admin)`
+      - Optional; required for using Eppo's [Event Tracking](/sdks/event-logging/event-tracking)
```

Comment:
I think we want to consolidate with the "Track API" naming which is easier to remember